From 902b760343a11f75593d3a1c54a07bf19eba5c0f Mon Sep 17 00:00:00 2001
From: Robert Chiras <robert.chiras@nxp.com>
Date: Thu, 3 May 2018 13:13:02 +0300
Subject: [PATCH 3661/5242] MLK-18193: arm64: dts: fsl-imx8mq: Fix DCSS clocks
 for panel use-case

commit  cbf1eac95db7a10aab605a0723b0e6ae5d49201f from
https://source.codeaurora.org/external/imx/linux-imx.git

The clocks for DCSS were incorrectly assigned in the rm67191 dts file,
so fix them.

Signed-off-by: Robert Chiras <robert.chiras@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts  |    5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts
index 19dbeb4..ed8d521 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-dcss-rm67191.dts
@@ -33,19 +33,18 @@
 	assigned-clocks = <&clk IMX8MQ_CLK_DC_PIXEL_SRC>,
 			  <&clk IMX8MQ_CLK_DISP_AXI_SRC>,
 			  <&clk IMX8MQ_CLK_DISP_RTRM_SRC>,
-			  <&clk IMX8MQ_CLK_DISP_RTRM_PRE_DIV>,
 			  <&clk IMX8MQ_VIDEO_PLL1_REF_SEL>,
+			  <&clk IMX8MQ_CLK_DISP_RTRM_PRE_DIV>,
 			  <&clk IMX8MQ_VIDEO_PLL1>;
 	assigned-clock-parents = <&clk IMX8MQ_VIDEO_PLL1_OUT>,
 				 <&clk IMX8MQ_SYS1_PLL_800M>,
 				 <&clk IMX8MQ_SYS1_PLL_800M>,
-				 <&clk IMX8MQ_VIDEO_PLL1_OUT>,
 				 <&clk IMX8MQ_CLK_25M>;
 	assigned-clock-rates = <600000000>,
 			       <800000000>,
 			       <400000000>,
-			       <400000000>,
 			       <0>,
+			       <400000000>,
 			       <599999999>;
 
 	dcss_disp0: port@0 {
-- 
1.7.9.5

